lua/tardis/parts/legacy/legacy_unused.lua

```diff
@@ -0,0 +1,21 @@
+--Legacy Interior - Unused Pull Handle
+
+local PART = {}
+PART.ID = "legacy_unused"
+PART.Name = "Legacy Unused Pull Handle"
+PART.Model = "models/drmatt/tardis/handbrake.mdl"
+PART.AutoSetup = true
+PART.Collision = true
+PART.Animate = true
+
+if SERVER then
+    function PART:Use(ply)
+        self:EmitSound("tardis/control_handbrake.wav")
+        local on = not self.interior:GetData("legacy-advanced",false)
```

Comment:
This legacy advanced thing should probably change the setting instead of having a separate compatibility layer "legacy-advanced" (that is cool though)